core/jazz_ui/src/app/pages/environment-overview/env-overview-section.component.html

```diff
@@ -258,8 +258,15 @@
 <div class="secret-container relative col-detail" *ngIf="isNotWebsite && tvaultEnabled">
 	<div *ngIf="!isSecretLoading && !isError">
 		<div class="secret-container__edit" [ngClass]="!editEvents? 'editMode':'nonEditMode'">
-			<div class="secret-container__edit--def"> Secret Management <img class="icons-style"
-					src="../../../assets/images/icons/iconmonstr-info-2.svg" /></div>
+			<div class="secret-container__edit--def"> Secret Management 
+				<div class="info-tooltip">
+					<img class="icons-style"
+						 src="../../../assets/images/icons/iconmonstr-info-2.svg" />
+						<div class="tooltip-content">
+							<span id="copyPopup">User can manage access to safe through these secrets. Click <a><i class="fa fa-clone" [style.fontSize]="iconSize"></i></a> to copy role</span>
```

Comment:
`User can manage access to safe through these secrets.` => `User can manage access to safe through these roles.`